### PR TITLE
increase sj collapsed limit

### PR DESCRIPTION
**Pull request by morsecodist**
We got a crash from this parameter so I doubled it.

### CODE REVIEW

File: short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py [hunk 1]
```diff
@@ -271,6 +271,9 @@ def run_star_part(self,
         #  when we update the rest of our rng seeds.
         rng_seed = '777'
 
+        # default is 1000000, this caused a crash so it was doubled
```
Comment (tfrcarvalho) [inline short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py:274]: Do we know what the parameter refers to?

Comment (morsecodist) [reply short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py:274]: `max number of collapsed junctions`

Comment (tfrcarvalho) [reply short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py:274]: ok... i guess does not had that much info without understanding the tool

**Review by tfrcarvalho — APPROVED** [review]

File: short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py (hunk 1 above)
Comment (morsecodist) [reply short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py:274]: From the manual, if you want more context: 

```
SJ.out.tabcontains high confidence collapsed splice junctions in tab-delimited format.  Note thatSTAR  defines  the  junction  start/end  as  intronic  bases,  while  many  other  software  define  them  asexonic bases.  The columns have the following meaning

column 1:  chromosome

column2:  first base of the intron (1-based)

column 3:  last base of the intron (1-based)

column 4:  strand (0:  undefined, 1:  +, 2:  -)

column 5:  intron  motif:  0:  non-canonical;  1:  GT/AG,  2:  CT/AC,  3:  GC/AG,  4:  CT/GC,  5:AT/AC, 6:  GT/AT

column 6:  0:  unannotated, 1:  annotated (only if splice junctions database is used)

column 7:  number of uniquely mapping reads crossing the junction

column 8:  number of multi-mapping reads crossing the junction

column 9:  maximum spliced alignment overhang

The filtering for this output file is controlled by the --outSJfilter parameters,  as described in Section 14.16.  Output Filtering:  Splice Junctions.
```

Comment (tfrcarvalho) [reply short-read-mngs/idseq-dag/idseq_dag/steps/run_star.py:274]: I was looking for just a more informative comment to add to the code, for the next (maybe on-call) person to do a quick decision of keeping to increase or just accept the error...

**Pull request merged**